Irrational slope topology (cut point, local metrizability, and more)

pzjp opened the pull request:
Adding traits (see #1074):
- P82 (loc. metrizable)
- not P89 (fixed point)
- P102 (semimetrizable) see #1075
- not P204 (cut point)
- proof for P44 (not biconnected), previously _assered in the general reference chart_.

Removing derivable traits:
- P37 (path connected)
- P22 (second countable)
- P27 (pseudocompact)
- P129 (indiscrete)

prabau commented on spaces/S000067/properties/P000082.md:
```diff
@@ -0,0 +1,16 @@
+
+The open subset $\mathbb Q{\times}\{0\}$ is homeomorphic to a metrizable space $\mathbb Q$.
+
+Consider $(x,y)\in X$ with $y>0$. Observe that for $\epsilon\in(0,y/\theta)$ the open neighbourhood $N_\epsilon(x,y)$ is homeomorphic to the following subset of $\mathbb R^2$ (hence it is metrizable):
```
```suggestion
Consider $(x,y)\in X$ with $y>0$. Observe that for $0<\epsilon<y/\theta$ the open neighbourhood $N_\epsilon(x,y)$ is homeomorphic to the following subset of $\mathbb R^2$ (hence it is metrizable):
```

prabau commented:
Nice observation for locally metrizable.

You are also careful enough to specify that $B(t,\varepsilon)$ consists of the rational numbers in a suitable interval.  The S&S book was also careful in its description, but the pi-base README for the space wasn't.

Could you for example modify the README and replace
"where $B(a, \epsilon)$ is an interval on the $x$-axis centered at $a$ with radius $\epsilon$"
with
"where $B(a, \epsilon)=\mathbb Q\cap (a-\epsilon,a+\epsilon)$" ?

prabau commented:
P102 (semimetrizable): I can see that for a point $z=(x,y)$ with $y>0$ the balls $B_d(z,\varepsilon)$ are the same as the $N_\varepsilon(z)$ (for $\varepsilon <1$).
But if $z=(t,0)$, neighborhoods of $z$ should be a small interval on the $x$-axis, but the balls $B_d(z,\varepsilon)$ seem to contain many points in the upper half plane, in diagonal strips going up with slopes $\theta$ and $-\theta$.  That does not seem to work?

prabau commented on spaces/S000067/properties/P000204.md:
```diff
@@ -0,0 +1,14 @@
+Suppose $A\cup B = X\setminus\{x\}$ for some $x\in X$ and disjoint nonempty open sets $A, B\subset X$ (note that {S67|P2}).
+In particular $\overline{A} \cap \overline{B}\subset (X\setminus B)\cap (X\setminus A)= \{x\}$,
+but the closures of any two nonempty open sets in $X$ have _infinite_ intersection
+(see space #75, property #2 in {{doi:10.1007/978-1-4612-6290-9_6}}).
```
```suggestion
(see item #2 for space #75 in {{doi:10.1007/978-1-4612-6290-9_6}}).
```

pzjp commented:
Good point. Maybe some modification resembling the construction for S60 could work...

prabau commented:
It does not seem trivial.  Would be worth asking the question on mathse, I think.

Of course, because the space is first countable, semimetrizable and symmetrizable would be equivalent in this case.

prabau commented:
FYI. Of the two remaining unknown traits, homogeneous is shown in https://zbmath.org/1453.54006.
It's not an easy result.

prabau commented:
Like you said, a construction similar to the one for S60 seems like it would work.  But checking this would be more work than what we usually do in a direct pi-base justification.  Worth a mathse post then.

yhx-12243 commented:
> FYI. Of the two remaining unknown traits, homogeneous is shown in https://zbmath.org/1453.54006. It's not an easy result.

This paper also implies the result that S60 is not P138 (irrational slope topology has uncountably-many continuous self-maps), by Theorem 1 we can extend arbitrary bijection of $\mathbb Z \to \mathbb Z$.

pzjp commented:
Posted the semimetrizability on mathse: [5019665](https://math.stackexchange.com/questions/5019665/is-the-bing-space-semimetrizable).
Should we refer to the name *Bing space* in the definition of S67?

pzjp commented:
Updated $B_\epsilon$ in the definition and changed P102 to a mathse link.

I added the reference to the original 1953 Bing's note defining the space. I see that pi-Base is not very concerned about the good citations (I doubt whether we should reach towards extensive bibliography either) but hopefully a single item is not a problem. I guess that if we know the first publication about a space it should be referenced alongside a possibly better, newer description. _Counterexamples_ themselves were not written in the way helping to find origins of the particular spaces.

Are we going to add homogenity and uncountable self-maps in this PR as well?

prabau commented:
FYI, _Counterexamples_ gives the full citation of the origin of this example, together with the modification by Ritter.  See p. 221 of the Notes section.

Will look at the rest later today.

prabau commented:
S67 README: you added an item to the `refs:` section.  What it is a paper or book, we put the author in parentheses at the end.  Compare with other pages.

prabau commented:
S67 README: not due to your change, but as phrased the description never says what the topology actually is.  Let me try to do an edit for that.

prabau commented:
P102: regarding you mathse post:

Referring to the "Bing space" in the title is not good.  Many spaces were invented by Bing; just saying Bing space is not helpful.  Furthermore, the paper by Banakh et al. called it the "Bing tripod space".  But nearly nobody on mathse, mo, or in the literature calls it by that name.  Nearly everybody knows that space by the name "Irrational space topology" (due to the influence of S&S, no doubt.  So I would recommend changing the title, and also the question.

(For the same reason, I would not refer to that space as "Bing space" in pi-base.  But adding the Bing reference, and now with the name of author, is perfect.)

Also, as a matter of policy, the "question part" of a mathse post should always have a question in the body itself.  So it should specifically ask if the irrational slope topology is semimetrizable.

Next, checking the answer part.

prabau commented:
I am not sure I understand the answer in your post.  Are you about the definition of $d(a,x)$ for $a\in G$ and $x\notin G$?
It seems it would work better with $p(x)$ replaced with $p(a)$ in that formula?

prabau commented:
As for homogeneity and uncountably many self maps, it's fine to do it now.  But up to you.

prabau commented on spaces/S000067/properties/P000086.md:
```diff
@@ -0,0 +1,10 @@
+  name: The connected countable spaces of Bing and Ritter are topologically homogeneous (Banakh et al.)
+---
+
+Homogeneity of $X$ (actually $n$-homegeneity, for any $n\in\mathbb N$) is proven in {{zb: 1453.54006}}.
```
```suggestion
Homogeneity of $X$ is proven in {{zb:1453.54006}}.
```
No need to mention more.

prabau commented on spaces/S000067/properties/P000102.md:
```diff
@@ -0,0 +1,10 @@
+value: true
+refs:
+- mathse: 5019665
+  name: Is the Bing space semimetrizable?
```
```suggestion
  name: Is the irrational slope topology semimetrizable?
```
new title

prabau commented:
Thanks for the fix for semimetrizability. Looks good.

prabau commented on spaces/S000067/properties/P000138.md:
```diff
@@ -0,0 +1,15 @@
+  name: The connected countable spaces of Bing and Ritter are topologically homogeneous (Banakh et al.)
+---
+
+$A\subset X$ is called *$\theta$-discrete* if every point of $X$ has a closed neighbourhood containing only one point of $A$. Symbol "$\theta$" in the name does not formally refer to to the irrational constant.
+
+By Theorem 1 in {{zb:1453.54006}}, every bijection between $\theta$-discrete subsets of $X$ extends to a homeomorphism of $X$.
+
+It can be readily verified that $Z=\mathbb Z\times\{0\}$ is $\theta$-discrete (every discrete closed subset contained in $\mathbb Q\times\{0\}$ is) and admits continuum bijections to itself (e.g. every subset of $Z$ can be the set of fixed points),
+hence there exist uncountably many self-homeomorphisms of $X$.
```
```suggestion
Following the terminology used in {{zb:1453.54006}}, a set
$A\subset X$ is called *$\theta$-discrete* if every point of $X$ has a closed neighbourhood containing only one point of $A$. (The symbol "$\theta$" here has nothing to do with the irrational constant used to define {S67}.)

By Theorem 1 in that paper, every bijection between $\theta$-discrete subsets of $X$ extends to a homeomorphism from $X$ to itself.

The set $\mathbb Z\times\{0\}$ is $\theta$-discrete (every discrete closed subset contained in $\mathbb Q\times\{0\}$ is) and admits continuum many bijections to itself (e.g. every subset of $Z$ with at least two elements in its complement can be the set of fixed points).
Hence there exist uncountably many self-homeomorphisms of $X$.
```

prabau commented:
As mentioned earlier, "Bing space" is not a good name for this space.
But what would you think of having "Bing's connected countable space" as an alias?

Comparing with "Ritter's connected countable space" as another interesting space that could possibly be added in the future

prabau commented:
Great.  Let me add the alias then.

pzjp commented:
I approve. So S67 is now completed with all traits!
